.github/config.yml

```diff
@@ -8,9 +8,9 @@ newPRWelcomeComment: >
 
   Note that translations should be made on [Crowdin](https://crowdin.com/project/electron) and _should not_ be submitted manually here as a pull request.
 
-  Please check out our [contributing guidelines](contributing.md).
+  Please check out our [contributing guidelines](https://github.com/electron/electron-i18n/blob/master/contributing.md).
```

Comment:
This is right.
The error is in [.github/config.yml](https://github.com/electron/electron-i18n/blob/bd5884edeaffa6b89cb4b2ee60fef2ebc993f614/.github/config.yml#L11) line 11.